_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,327 @@
+```
+
+##### Explanation:
+This happens when the GKE cluster is not able to schedule pods when Terraform attempts to deploy Forseti on the cluster via Helm.  This is to be addressed via a patch in a future version of the [terraform-google-kubernetes-engine](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine) module.
+
+##### Workaround:
+Re-run `terraform apply`
+```
+terraform apply
+```
+<br />
+#### Terraform Destroy - Error, failed to deleteuser root 
+
+##### Error Message:
+```bash
+Error: Error, failed to deleteuser root in instance forseti-server-db-1914b4e1: googleapi: Error 400: Missing parameter: host., required
+```
+
+##### Explanation:
+This is under investigation.  Please see forseti-security/forseti-security GitHub [issue #148](https://github.com/forseti-security/terraform-google-forseti/issues/148).  
+
+##### Workaround:
+```bash
+terraform state rm module.forseti-on-gke-end-to-end.module.forseti.module.server.google_sql_user.root
+```
+
+<br />
+#### Terraform Destroy - could not find a ready tiller pod
+
+##### Error message:
+```bash
+Error: error creating tunnel: "could not find a ready tiller pod"
+```
+
+##### Explanation:
+This occurs if the GKE cluster is destroyed before Terraform destroys the Helm release.  This will be resolved in the near future.  While this issue has been resolved via terraform-google-modules/terraform-google-kubernetes-engine [pull request #214](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/214), this documentation will remain for reference.
+
+##### Workaround:
+```bash
+terraform state rm module.forseti-on-gke-end-to-end.module.forseti-on-gke.helm_release.forseti-security
+
+terraform state rm module.forseti-on-gke-end-to-end.module.forseti-on-gke.kubernetes_namespace.forseti
+```
+
+<br />
+#### Terraform Destroy - Cannot delete auto subnetwork
+
+##### Error Message:
+```bash
+Error: Error reading Subnetwork: googleapi: Error 400: Invalid resource usage: 'Cannot delete auto subnetwork from an auto subnet mode network.'., invalidResourceUsage
+```
+
+##### Explanation:
+This occurs when there are resources on the subnetwork that exist outside of Terraform's state.  For instance, Forseti may have been deployed in a separate run of Terraform.
+
+##### Workaround:
+```bash
+terraform state rm module.forseti-on-gke-new-gke-cluster.module.vpc.google_compute_network.network
+
+terraform state rm module.forseti-on-gke-new-gke-cluster.module.vpc.google_compute_subnetwork.subnetwork[0]
+```
+#### Terraform - Help, my error isn't listed here.
+There are a few things you can do.
+1. Run `terraform apply` or `terraform destroy` again to see if the error occurs repeatedly.
+2. Open an issue against the [Terraform Google Forseti](https://github.com/forseti-security/terraform-google-forseti/issues) module
+3. Post an issue in our [Slack Channel](forsetisecurity.slack.com).  We'll be happy to try and help!
```

Comment:
Should change (forsetisecurity.slack.com) --> (https://forsetisecurity.slack.com/join/shared_invite/enQtNDIyMzg4Nzg1NjcxLTJhYjI1YzY0MDg4YjRhMDhhZTMwZTg0MWExZjU1MTNiNWFmMmFlMWQ0MmI3OTE1MzczZTMwNTEzNDZiNDY3NTY)